evalml/utils/gen_utils.py

```diff
@@ -514,3 +514,38 @@ def save_plot(fig, filepath=None, format='png', interactive=False, return_filepa
 
     if return_filepath:
         return filepath
+
+
+class suppress_stdout_stderr(object):
```

Comment:
Prophet doesn't have a flag for verbosity so we need this context manager to remove their training output:

https://github.com/facebook/prophet/issues/223

Reply:
@freddyaboulton  the sktime source grabbed it from the issue so I think linking the issue is fine here

https://github.com/facebook/prophet/issues/223#issuecomment-326455744